[33m[W 2025-05-13 19:02:22,343][0m Trial 0 failed with parameters: {'lr': 6.854442896192116e-05, 'weight_decay': 0.00040312115024765833, 'batch_size': 64, 'n_layers': 3, 'hidden_dim': 1024} because of the following error: AttributeError("'TargetComputer' object has no attribute 'norm'").[0m
Traceback (most recent call last):
  File "/home/panda/miniconda3/lib/python3.12/site-packages/optuna/study/_optimize.py", line 197, in _run_trial
    value_or_values = func(trial)
                      ^^^^^^^^^^^
  File "/home/panda/Soccer/train.py", line 477, in <lambda>
    lambda t: _objective(t, train_paths, val_paths, epochs, target_type),
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/panda/Soccer/train.py", line 463, in _objective
    trainer.train()
  File "/home/panda/Soccer/train.py", line 396, in train
    tr_loss = self._epoch(self.train_loader, True)
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/panda/Soccer/train.py", line 370, in _epoch
    targets = self.target_computer.compute(batch, self.q_net, self.device)
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/panda/Soccer/train.py", line 169, in compute
    return (raw_targets - self.norm.G_mean) / self.norm.G_std
                          ^^^^^^^^^
AttributeError: 'TargetComputer' object has no attribute 'norm'
[33m[W 2025-05-13 19:02:22,344][0m Trial 0 failed with value None.[0m
Traceback (most recent call last):
  File "/home/panda/Soccer/train.py", line 503, in <module>
    run_optuna(args.shards, args.trials, args.epochs, args.target)
  File "/home/panda/Soccer/train.py", line 476, in run_optuna
    study.optimize(
  File "/home/panda/miniconda3/lib/python3.12/site-packages/optuna/study/study.py", line 475, in optimize
    _optimize(
  File "/home/panda/miniconda3/lib/python3.12/site-packages/optuna/study/_optimize.py", line 63, in _optimize
    _optimize_sequential(
  File "/home/panda/miniconda3/lib/python3.12/site-packages/optuna/study/_optimize.py", line 160, in _optimize_sequential
    frozen_trial = _run_trial(study, func, catch)
                   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/panda/miniconda3/lib/python3.12/site-packages/optuna/study/_optimize.py", line 248, in _run_trial
    raise func_err
  File "/home/panda/miniconda3/lib/python3.12/site-packages/optuna/study/_optimize.py", line 197, in _run_trial
    value_or_values = func(trial)
                      ^^^^^^^^^^^
  File "/home/panda/Soccer/train.py", line 477, in <lambda>
    lambda t: _objective(t, train_paths, val_paths, epochs, target_type),
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/panda/Soccer/train.py", line 463, in _objective
    trainer.train()
  File "/home/panda/Soccer/train.py", line 396, in train
    tr_loss = self._epoch(self.train_loader, True)
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/panda/Soccer/train.py", line 370, in _epoch
    targets = self.target_computer.compute(batch, self.q_net, self.device)
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/panda/Soccer/train.py", line 169, in compute
    return (raw_targets - self.norm.G_mean) / self.norm.G_std
                          ^^^^^^^^^
AttributeError: 'TargetComputer' object has no attribute 'norm'
